functorch/_src/partitioners.py

```diff
@@ -207,7 +207,7 @@ def _count_ops(graph):
 
 
 def min_cut_rematerialization_partition(
-    joint_module: fx.GraphModule, _joint_inputs, compiler="nvfuser"
+    joint_module: fx.GraphModule, _joint_inputs, compiler="nvfuser", recomputable_ops=None,
```

Comment:
Should we just unify these two kwargs? If `recomputable_ops == "nvfuser" or "inductor"`, then use the appropriate list. Otherwise, allow for a fully custom list?

Otherwise, this doesn't allow for removing ops from the list.

Reply:
If this `recomputable_ops` is not None then it overwrites whatever list is there for "nvfuser" or inductor" defined below.

I didn't want to touch "compiler" argument because it would need a corresponding change in Inductor's code.